### ISSUE TITLE
Numbers as keys results in error

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#390]
# Language
JavaScript

# Sample input source code
```
const obj = {
  2: 'b',
  1: 'a',
};
```

# Expected output
```
const obj = {
  1: 'a',
  2: 'b',
};
```

# Actual output (or error message)
Some error... awaiting results

**Comment by k2snowman69** [comment]
Likely the best place to start investigating this is to add a unit test here:
https://github.com/snowcoders/sortier/tree/master/src/language-js/sortObjectTypeAnnotation/test_assets

**Comment by k2snowman69** [comment]
Released at 2.5.4
